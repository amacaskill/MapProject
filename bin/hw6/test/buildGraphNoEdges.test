LoadGraph noEdges buildGraphNoEdges.tsv

ListNodes noEdges 

ListChildren noEdges char1
ListChildren noEdges char2
ListChildren noEdges char3