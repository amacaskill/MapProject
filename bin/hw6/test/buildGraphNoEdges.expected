loaded graph noEdges

noEdges contains: char1 char2 char3

the children of char1 in noEdges are:
the children of char2 in noEdges are:
the children of char3 in noEdges are:
